asv/www/regressions.js

```diff
@@ -364,11 +366,11 @@ $(document).ready(function() {
         if (regression === null) {
             return null;
         }
-        var dates = regression[0];
+        var revisions = regression[0];
         var ignore_payload = benchmark_name;
 
-        $.each(dates, function (i, date) {
-            ignore_payload = ignore_payload + ',' + date[0] + ',' + date[1];
+        $.each(revisions, function (i, revs) {
+            ignore_payload = ignore_payload + ',' + revs[0] + ',' + revs[1];
```

Comment:
It's fine to change the hashes, as things become just un-ignored --- this is a per-user convenience feature (doesn't transfer between different browsers etc.).